Shared/Flows/Launch/SplashViewController.swift

```diff
@@ -13,8 +13,22 @@ import Localization
 
 class SplashViewController: FullScreenViewController, TransitionableViewController {
 
-    var receivingPresentationType: TransitionType {
-        return .fade
+    // MARK: - Transitionable
+    
+    var presentationType: TransitionType {
+        return .fadeOutIn
+    }
+
+    var dismissalType: TransitionType {
+        return self.presentationType
+    }
+
+    func getFromVCPresentationType(for toVCPresentationType: TransitionType) -> TransitionType {
+        return toVCPresentationType
+    }
+
+    func getToVCDismissalType(for fromVCDismissalType: TransitionType) -> TransitionType {
+        return fromVCDismissalType
```

Comment:
For now, all of these var/functions need to be implemented by transitionable VCs. We can add them to the ViewController base class if we want default behavior.